.github/workflows/deploy-strapi.yml

```diff
@@ -0,0 +1,42 @@
+name: Build Strapi on Deploy
+
+on:
+   workflow_dispatch:
```

Comment:
Just the presence of the key allows you to manually fire this job, otherwise the only way to fire it is by pushing a commit.

> When using the workflow_dispatch event, you can *optionally* specify inputs that are passed to the workflow.

They even have this as an example:
![image](https://user-images.githubusercontent.com/26855/231812554-66ccdf59-25b2-418b-bb5e-004fcaf491a6.png)

Reply:
Oh interesting. How does one manually fire a job? It's not obvious to me from the UI or the docs. I'm curious if we should add this key to our other workflows too.

Reply:
For workflows that have it we see this:
![image](https://user-images.githubusercontent.com/26855/231815871-53c190ed-849e-4cf2-9692-837247516060.png)

like this:
https://github.com/iFixit/ifixit/actions/workflows/critical-css.yml